package/firmware/ipq-wifi/Makefile

```diff
@@ -40,13 +40,19 @@ ALLWIFIBOARDS:= \
 	wallys_dr40x9 \
 	xiaomi_ax3600 \
 	xiaomi_ax9000 \
+	yuncore_ax880 \
 	zte_mf289f \
 	zte_mf287 \
 	zte_mf287plus \
 	zyxel_nbg7815
 
 ALLWIFIPACKAGES:=$(foreach BOARD,$(ALLWIFIBOARDS),ipq-wifi-$(BOARD))
 
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	cp board/* $(PKG_BUILD_DIR)
+endef
+
```

Comment:
Why did you add this?

Reply:
Due to the lack of the correct BDF file for the device in the corresponding repo (qca-wireless).

I posted the BDF file to the ath11k mailing list. Until the file for the device is added to the related repo, it is located here. It is copied in addition to the others, which are downloaded from the qca-wireless repo.

Reply:
I'm talking about ```Build/Prepare``` section...

Reply:
8-) 


This recipe copies the BDF file for the Yuncore AX880 device (https://github.com/openwrt/openwrt/blob/9297dfcc4d6263612f6b399be7f043ea4daa88cb/package/firmware/ipq-wifi/board/board-yuncore_ax880.ipq8074) to the qca-wireless package directory, from where this file is later copied to the compiled firmware with this recipe `ipq-wifi-install-ath11-one-to` (https://github.com/openwrt/openwrt/blob/9297dfcc4d6263612f6b399be7f043ea4daa88cb/package/firmware/ipq-wifi/Makefile#L69).

Reply:
Yeah, this should be dropped as we are intentionally shipping the BDF-s in a separate repo.
I think I saw you sent the BDF as a patch instead of a PR on GH?

Reply:
If i remove this recipe before related BDF will be accepted automatic check will fail. What to do?